Gopkg.toml

```diff
@@ -32,29 +32,24 @@ required = [
 
 [[constraint]]
   name = "github.com/rs/cors"
-  version = "1.2.0"
+  version = "1.7.0"
 
 [[constraint]]
   name = "github.com/sirupsen/logrus"
-  version = "1.0.3"
+  version = "1.7.0"
 
 [[constraint]]
   name = "github.com/spf13/pflag"
-  version = "1.0.0"
+  version = "1.0.5"
 
 [[constraint]]
   name = "github.com/stretchr/testify"
-  version = "1.1.4"
+  version = "1.7.0"
 
 [[constraint]]
   branch = "master"
   name = "golang.org/x/net"
 
 [[constraint]]
   name = "google.golang.org/grpc"
-  version = "1.8.0"
-
-# To work around github.com/grpc-ecosystem/go-grpc-middleware requiring master
-[[override]]
-  name = "github.com/golang/protobuf"
-  version = "1.3.0"
+  version = "1.35.0"
```

Comment:
FYI this version of gRPC depends on the APIv2 of Go protobufs: https://github.com/grpc/grpc-go/blob/v1.35.x/go.mod#L16. We may want to lock to [1.32.x](https://github.com/grpc/grpc-go/blob/v1.32.x/go.mod#L9) which seems to be the latest major using APIv2.